client/src/index.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+ReactDOM.render(
+  <BrowserRouter>
```

Comment:
Create a separate route file for routes and i.port from that. As there will be many routes

Reply:
ok, I will make the seprate router file